tasks/set_masquerade.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: flush iptables
+  shell: sudo iptables --flush
+
+- name: get default route
+  shell: |
+    sudo ip r | grep default | cut -d ' ' -f5
+  register: default_route
+
+- name: masquerade on public interface
+  shell: |
+    sudo iptables -t nat -A POSTROUTING -o {{ default_route.stdout }} -j MASQUERADE
```

Comment:
`sudo iptables --flush` will not flush rules in the nat table, as far as I have observed.
Is it required to add this rule again here? It will lead to the same rule being added multiple times.

Reply:
This rule will not be added multiple times, we can customize in shift_stack_vars now. 
https://github.com/redhat-performance/jetpack/pull/485/commits/e501c7f4d1cdc1e4959d6bff12c4afa63a33d6bf#diff-f9144ea1aa91b6992a70c42069fbd7414958b637270934198cbf3d5d2a25681aR17-R18

Reply:
> `sudo iptables --flush` will not flush rules in the nat table, as far as I have observed.

Even I see that rules in nat are not flushed with this. but the old playbook[1] has this task.. so i preserved the same
[1] https://github.com/redhat-performance/jetpack/blob/master/ocp_on_osp.yml#L11

Reply:
Ack